ckan/logic/action/get.py

```diff
@@ -726,9 +733,18 @@ def organization_list_for_user(context, data_dict):
             (org, group_ids_to_capacities[org.id]) for org in orgs_q.all()]
 
     context['with_capacity'] = True
-    orgs_list = model_dictize.group_list_dictize(orgs_and_capacities, context,
-        with_package_counts=asbool(data_dict.get('include_dataset_count')))
-    return orgs_list
+    validated_organizations = []
+    for org, capacity in orgs_and_capacities:
+        validated_org_and_capacity = logic.get_action('organization_show')(
+            plugins.toolkit.fresh_context(context),
+            {'id': org.id,
+            'include_dataset_count': asbool(data_dict.get('include_dataset_count'))})
```

Comment:
this is going to be much slower for e.g. sysadmin users that belong to all orgs. Let's do all the `include...=False` stuff here too